es-ds-docs/pages/molecules/data-table.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div>
+        <h1>
+            Data table
+        </h1>
+        <p class="mb-500">
+            A responsive table using the <nuxt-link to="https://primevue.org/datatable/">PrimeVue DataTable</nuxt-link>.
```

Comment:
Nit: these URLs should be https://v3.primevue.org/datatable/ since we're still on v3